### ISSUE TITLE
Request: Improve black and white output quality

### ISSUE THREAD

**Issue by rudolphos** [4lex4/scantailor-advanced#169]
I'm only using ScanTailor for deskewing and margins as those are the best features this software has to offer, but most of the time black-and-white text output quality is bad and too noisy, it removes too much from the original and creates some parts of a character missing.

Original
![image](https://user-images.githubusercontent.com/4366810/111014853-6c552680-83ae-11eb-8d09-520925c8537f.png)

This is how an output looks in ABBYY FineReader using just 'Whiten background' feature with no other options. I suspect it just uses grayscale and posterize? Since one page tif export is around the same size (80 KB) as on ST.
![image](https://user-images.githubusercontent.com/4366810/111014790-2009e680-83ae-11eb-919f-06e1e9cf96e3.png)

Here's ScanTailor output with default settings
![image](https://user-images.githubusercontent.com/4366810/111014897-b1795880-83ae-11eb-9b10-7d2eb2fcb316.png)

Savitzky, Morph. disabled:
![image](https://user-images.githubusercontent.com/4366810/111014882-9870a780-83ae-11eb-9852-2492f95a5ea1.png)

ABBYY had the result most similar to the original text.

**Comment by mara004** [comment]
You're right that the ScanTailor output is not exactly as precise and smooth as the ABBY output, but overall it seems fairly similar to me.

**Comment by ftrebien** [comment]
I've always wanted to delve deeper into how DjVu does this (see the sixth example [here](https://djvu.sourceforge.net/screenshots.html)). Meanwhile, when I want to remove the background from a dirty scanned page, I do the following in GIMP:
1. Open an image and duplicate its single layer
2. Estimate the background color by applying a median blur filter to the new layer with a percentile above 50% (usually 70%) adjusting the radius to the content (usually 80px at 300dpi or more if there are large illustrations, until the preview has no blobs related to actual content)
3. Remove the estimated background inverting the colors of the new layer, setting the layer mode to Addition (Legacy) and merging it with the original layer

As this requires previewing to determine the best blur radius, it cannot be applied automatically without risking destroying some content. But a manual adjustment may be compatible with ScanTailor's UI. This works great with these "default" settings for text-only documents with no graphics other than thin lines.

Sometimes even a very large radius can still leave some content-related blobs. In these cases, between steps 2 and 3, I select the remaining blobs and repeat the median blurring to reduce them further, which fills them with the surrounding background color, and then I apply a bit of Gaussian blur blurring to soften the edges of the image. selection. But all this manual work may not suit ScanTailor's UI.

After removing the background, I apply GIMP White Balance to make the text very readable.
